Readme.md

```diff
@@ -96,3 +96,19 @@ you change, you may have to reload the extension in your browser:
   Reload.
 - If you changed the 'content_script', it seems browser-dependent whether newly
   opened pages will get the new version. Better reload the extension to be sure.
+
+### Guideline for Contribution
+
+Please follow the given guideline to make it easier for the maintainer to review and evaluate your work.
+
+**NOTE:**By sending a pull request you willingly waive all your copyrights on the code, so that it can be published in the public domain. For further information look into [Unlicense](http://unlicense.org/)
+
+* It is recommended that the developers use a text editor that supports editor config files for consistent spacing and other code styling.
+
+* Before taking on an issue please outline your approach in the corresponding issue. This way we can pool all of our ideas and come up with the best solution together. 
+
+* Before sending a pull request, make sure the code conforms to the styling guide. You can check for errors in styling by running the command ```sudo npm run lint```. For trivial error fixes it is recommended that you use ```sudo npm run lint-fix```.
```

Comment:
There's no reason to be recommending the user to run NPM with `sudo`, especially for scripts in a dev project. Check [fixing npm permissions docs](https://docs.npmjs.com/getting-started/fixing-npm-permissions), if you're having issues forcing you to run `npm` with `sudo` all the time.